<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1532</width>
    <height>923</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="setup_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>SETUP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="go_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="N_trees_spinBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QChartView" name="main_map">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="species_ratio_spinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
   </widget>
   <widget class="QChartView" name="N_birch_pop_chart">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="N_oak_pop_chart">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>40</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="N_years_spinBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of years</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of trees 
per hectare</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Birch to oak ratio 
0 = only birch</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sim_fire_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>simulate forest fire?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="deadwood_removed_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>deadwood removed?</string>
    </property>
   </widget>
   <widget class="QChartView" name="N_birch_burnt_area_chart">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>350</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="N_oak_burnt_area_chart">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>350</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="progress_output_textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress output</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="burnt_area_radius_spinBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius of burnt area [m]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>red:   birch trees 
blue: oak trees 

green: seedling and sapling density 

black: burnt area</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1532</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuForest_fire_succession_model">
    <property name="title">
     <string>Forest fire succession model</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFlorian_Wilms_M_Sc_Forest_Ecosystem_Sciences_Ecosystem_Analysis_and_Modelling">
    <property name="title">
     <string>Florian Wilms | M.Sc. Forest Ecosystem Sciences - Ecosystem Analysis and Modelling</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEcological_Modelling_wit_C_Winter_Semester_2023_24">
    <property name="title">
     <string>Ecological Modelling with C++ | Winter Semester 2023/24</string>
    </property>
   </widget>
   <addaction name="menuForest_fire_succession_model"/>
   <addaction name="menuFlorian_Wilms_M_Sc_Forest_Ecosystem_Sciences_Ecosystem_Analysis_and_Modelling"/>
   <addaction name="menuEcological_Modelling_wit_C_Winter_Semester_2023_24"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
